ccd-definition/AuthorisationCaseField/CareSupervision/la-solicitor.json

```diff
@@ -2485,5 +2485,19 @@
     "CaseFieldID": "sendPlacementNoticeToAllRespondents",
     "UserRole": "[LASOLICITOR]",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "UserRole": "[LASOLICITOR]",
+    "CaseFieldID": "refusedOrdersTabHeading",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "UserRole": "[LASOLICITOR]",
+    "CaseFieldID": "refusedHearingOrders",
+    "CRUD": "CRU"
```

Comment:
Do we also need to extend this permission to the Managing LAs / External solicitors managing the case on behalf of the LA (just thinking of the LA which outsources **everything**)